docs/tutorials/quantum-approximate-optimization-algorithm.ipynb

```diff
@@ -44,7 +44,9 @@
     "\n",
     "Before starting this tutorial, be sure you have the following installed:\n",
     "- Qiskit SDK v1.0 or later, with visualization support ( `pip install 'qiskit[visualization]'` )\n",
-    "- Qiskit Runtime 0.22 or later (`pip install qiskit-ibm-runtime`)"
+    "- Qiskit Runtime 0.22 or later (`pip install qiskit-ibm-runtime`)\n",
+    "\n",
+    "In addition, you will need access to an instance on the [IBM Quantum Platform](https://quantum.cloud.ibm.com/docs/en/guides/cloud-setup). Note that this notebook cannot be executed on the free tier, as [Qiskit Sessions](https://quantum.cloud.ibm.com/docs/en/api/qiskit-ibm-runtime/session) are available only with a commercial plan."
```

Comment:
```suggestion
    "In addition, you will need access to an instance on [IBM Quantum Platform](/docs/guides/cloud-setup). Note that this tutorial cannot be executed on the Open Plan, because it runs workloads using [sessions](https://quantum.cloud.ibm.com/docs/en/api/qiskit-ibm-runtime/session), which are only available with Premium Plan access."
```